treescript/requirements/base.txt

```diff
@@ -201,9 +205,9 @@ requests==2.22.0 \
     --hash=sha256:11e007a8a2aa0323f5a921e9e6a2d7e4e67d9877e85773fba9ba6419025cbeb4 \
     --hash=sha256:9cf5292fcd0f598c671cfc1e0d7d1a7f13bb8085e9a590f48c010551dc6c4b31 \
     # via github3.py, taskcluster
-scriptworker==29.0.0 \
-    --hash=sha256:32d06bedaa7f8134cdb6319ff4c1d044c03472ba3145b971989a52527858b274 \
-    --hash=sha256:91f999f3eb60f7fd43b5e8df3f260ed79d277a930823a33de3618692d90ba89b
+scriptworker==29.0.1 \
```

Comment:
Hm. I need 29.0.2 on treescript. (I just released that for https://bugzilla.mozilla.org/show_bug.cgi?id=1597671 ).
We can push your change, and then I can push my change to production-treescript, or we can push once with 29.0.2. I'm running pin.sh right now.